buildenv/jenkins/pipeline-functions

```diff
@@ -51,6 +40,13 @@
     }
 }
 
+def get_sha(REPO, BRANCH) {
```

Comment:
Can you comment this to describe what it does and why it passes both repo & branch?

Reply:
Pushed new commit with some comments